catalog/USGS/USGS_NLCD_RELEASES_2020_REL_NALCMS.jsonnet

```diff
@@ -0,0 +1,399 @@
+local id = 'USGS/NLCD_RELEASES/2020_REL/NALCMS';
+local subdir = 'USGS';
+local version = '1';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+local units = import 'units.libsonnet';
+
+local license = spdx.cc0_1_0;
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  // TODO(schwehr): Remove skip_indexing when the preview image is ready.
+  'gee:skip_indexing': true,
+  'gee:user_uploaded': true,
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+    ee_const.ext_ver,
+  ],
+  id: id,
+  title: 'Land Cover of North America at 30 meters, 2020',
+  version: '1',
+  'gee:type': ee_const.gee_type.image,
+  description: |||
+    The 2020 North American Land Cover 30-meter dataset was produced as part of
+    the North American Land Change Monitoring System (NALCMS), a trilateral
+    effort between Natural Resources Canada, the United States Geological
+    Survey, and three Mexican organizations including the National Institute of
+    Statistics and Geography (Instituto Nacional de Estadistica y Geografia),
+    National Commission for the Knowledge and Use of the Biodiversity (Comision
+    Nacional Para el Conocimiento y Uso de la Biodiversidad), and the National
```

Comment:
Accents:
Instituto Nacional de Estadística y Geográfia
Comisión Nacional Para el Conocimiento y Uso de la Biodiversidad
Comisión Nacional Forestal

Reply:
Intentionally removed in https://github.com/google/earthengine-catalog/pull/617/commits/02e7314d03186cf4048747cbafc9cd1806a88cbf

Doing some follow up checking, but I'm pretty sure we need to solve this a different way

Reply:
Working on it in https://github.com/google/earthengine-catalog/pull/619